src/components/Ballot.js

```diff
@@ -1,57 +1,57 @@
-import React, { Component } from 'react';
-import Category from './Category';
-import SubmitButton from './SubmitButton';
-import oscarsJSON from '../api/oscars.json';
-import { prop } from 'ramda';
-
-
+import React, { Component } from 'react'
+import { prop } from 'ramda'
+import Category from './Category'
+import SubmitButton from './SubmitButton'
+import oscarsJSON from '../api/oscars.json'
 
 class Ballot extends Component {
   // TODO this is deprecated
-  componentWillMount() {
-    // this.checkAuth();
-  }
+  // componentWillMount() {
+  //   this.checkAuth();
+  // }
 
   checkAuth() {
-    const { router, user } = this.props;
+    const { router, user } = this.props
     if (!user.isLogged) {
-      router.push('/');
+      router.push('/')
```

Comment:
This I don't like it. 
Sometimes you can concatenate functions like 

`
.then()
.then()
`
or

.map()
.somethingElse()
`

I think is cool to know that there is no ; behind because is not the end of the expression.